src/schema/common.ts

```diff
@@ -47,13 +46,6 @@ export const ReceiveFramePayload = zod
   })
   .passthrough();
 
-export const ScopesObject = {
-  ...AuthenticateResponseSchema.shape.scopes.element.overlayType._def.innerType.options[0].Values,
-  UNHANDLED: -1,
-} as const;
-
-export const Scopes = zodCoerceUnhandledValue(ScopesObject);
-
```

Comment:
We need to be careful about removing exports. Depending on dev usage, this could break the package for devs

Reply:
I think they should use `Types.OAuthScopes` instead. is there a way to suggest this?